node/src/Cardano/Node/API.hs

```diff
@@ -30,11 +33,38 @@ import qualified Pos.DB.Rocks.Types as DB
 import           Pos.Infra.Diffusion.Subscription.Status (ssMap)
 import           Pos.Infra.Diffusion.Types
 import qualified Pos.Infra.Slotting.Util as Slotting
+import           Pos.Launcher.Resource
 import           Pos.Node.API as Node
 import           Pos.Util (HasLens (..), HasLens')
 import           Pos.Util.CompileInfo (CompileTimeInfo, ctiGitRevision)
 import           Pos.Util.Servant
 
+launchNodeServer
+    :: Diffusion IO
+    -> NtpConfiguration
+    -> NodeResources ext
+    -> UpdateConfiguration
+    -> CompileTimeInfo
+    -> IO ()
+launchNodeServer diffusion ntpConfig nodeResources updateConfiguration compileTimeInfo = do
+    ntpStatus <- withNtpClient (ntpClientSettings ntpConfig)
+    let app = serve (Proxy @Node.API)
+            $ handlers
+                diffusion
+                ntpStatus
+                (ncStateLock nodeCtx)
+                (nrDBs nodeResources)
+                (ncLastKnownHeader nodeCtx)
+                slottingVarTimestamp
+                slottingVar
+                updateConfiguration
+                compileTimeInfo
+
+    Warp.run 3000 app
```

Comment:
The node already listens on `tcp/3000` for its node <-> node protocol. We gotta get that from the CLI as we do for the wallet, with a sensible default (happy to have that in another PR)

Reply:
Ok, cool! I'll make a copy of the `WalletBackendParams` type for `NodeBackendParams` and omit the stuff we don't need from it.